aws/modules/register/elb.tf

```diff
@@ -2,7 +2,7 @@ resource "aws_elb" "load_balancer" {
   count = "${signum(var.instance_count)}"
 
   # replace discovery with disco to ensure name does not exceed string length limit
-  name = "${replace(format("%s-%s", replace(var.vpc_name,"discovery","disco"), var.id),"_","-")}"
+  name = "${replace(format("%s--%s", replace(var.vpc_name,"discovery","disco"), var.id),"_","-")}"
```

Comment:
Is there a reason for having the extra dash? Is it because you rebuilt the world whilst always having a load balancer running?